docs/challenge/index.rst

```diff
@@ -0,0 +1,108 @@
+The final Pull Request submission date and time must take place before **July 13, 2023 at 16:59 (Pacific Standard Time)**.
+Participants are welcome to modify their Pull Request until this time.
+
+How to Submit
+-------------
+Everyone can participate and participation is free. It is sufficient to:
+
+- send a Pull Request
+- respect Submission Requirements
+
+An acceptable Pull Request automatically subscribes a participant/team to the challenge.
+
+Guidelines
+-------------
+We encourage the participants to start submitting their Pull Request early on. This helps debug failing tests and helps address potential issues with the code.
+
+In the case of multiple submissions implementing the same model, earlier Pull Requests will be given priority consideration.
+
+Teams are accepted and there is no restriction on the number of team members.
+
+One model per Pull Request. There is no restriction on the amount of submissions (Pull Requests) per participant/team. 
+
+The principal developers of TopoModelX are not allowed to participate.
+
+Submission Requirements
+-------------
+The submission must implement a pre-existing model from the literature included in Fig. 11 of the review `Architectures of Topological Deep Learning: A Survey of Topological Neural Networks <https://arxiv.org/pdf/2304.10031.pdf>`_.
+
+All submitted code must comply with TopoModelX's GitHub Action workflow, successfully passing all tests, linting, and formatting (i.e. Black, isort, flake8).
+
+The submission consists of a Pull Request to TopoModelX that contains three new files:
+
+1. {name of model}_layer.py (ex.: hsn_layer.py) :
+
+- stored in the directory topomodelx/nn/{domain of model}, where {domain of model} is simplicial, cellular, or hypergraph.
+- contains one class, {Name of model}Layer (ex.: HSNLayer), which uses TopoModelX computational primitives to implement one layer of the model. One layer is equivalent to the message passing depicted in the tensor diagram representation for the model (Fig. 11, Architectures of Topological Deep Learning).
+- examples are provided in topomodelx/nn/simplicial and topomodelx/nn/cellular. 
+
+2. {name of model}_train.ipynb (ex.: hsn_train.ipynb) :
+
+- stored in the directory tutorials/ and contains the following steps:
+
+  1. Pre-processing
+        - imports necessary packages as well as {Name of model}Layer class
+        - loads the protein-protein-interaction dataset `using TopoNetX <https://github.com/pyt-team/TopoNetX/blob/71e840ea5a475027ca9b4231563834547463cf19/toponetx/datasets/utils.py#LL9C6-L9C6>`_ and assigns labels.
+        - lifts the dataset from the graph domain to the topological domain of choice (hypergraph, simplicial complex, cellular complex, combinatorial complex) using TopoNetX.
+  
+  2. Creating the neural network
+        - defines a class {Name of model} (ex.: HSN) that inherits from torch.nn.Module and uses {Name of model}Layer along with torch.Linear layers to create a Topological Neural Network.
```

Comment:
NIT: backquotes: ``text`` for code samples in .rst

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html